### ISSUE TITLE
LESS files are missing from source

### ISSUE THREAD

**Issue by jeffreyselby** [WestpacCXTeam/GUI-source#132]
LESS files appear to be missing from ZIP file downloaded from Blender.

When using the Blender and selecting "Include the LESS files in your blend?" not all LESS files are made available. I was expecting more files to be included in the downloaded folder:
GUI-blend-WBC\GUI-blend\source\less

According to me the following LESS files are missing:

icons-group01.less
icons-group02.less
icons-group03.less
icons-group04.less
icons-group05.less
icons-group06.less
icons-group07.less
icons-group08.less
icons-group09.less
icons-group10.less
icons-group11.less
icons-group12.less
icons-group13.less
icons-group14.less
icons-group15.less
icons-group16.less

Could you please confirm these files are missing from the download and if so, please make them available.

Many Thanks,

Jeff

**Comment by dominikwilkowski** [comment]
Hi Jeff,

Thanks for reporting this. The thing is though there is no less in these modules to be included.
The icon modules consist only of svgs so that's more or less it, excuse the pun.

Let me know if you have any other issues :)

**Comment by dominikwilkowski** [comment]
I was too quick to answer here eager to get a pun in ;)

You are right actually that the `icons-group01` module should include less as it is a css animation.

Will fix now.

**Comment by jeffreyselby** [comment]
Hi Dominik,

At first I thought the LESS files were not required, but I noticed that a class such as "icon-loading-inverse" is defined in gui.min.css. 

When I generate the CSS file, that class is not there in my output. 

Just saw your last comment. Thanks for addressing this issue.

Regards,

Jeff

**Comment by jeffreyselby** [comment]
Thanks Dominik. One other thing is that should the Blender also make a gui.less file available.

I manually created such a file to reference all other LESS files, but it might be handy if a parent LESS file is made available from blender.

**Comment by jeffreyselby** [comment]
Also Dominik, when an issue is marked as closed and committed, when will the Blender be updated such that the new LESS files will be available in the download? Is this immediate integration or does a release have to be planned to update the Blender?

Had a look on GitHub and could not find the files icon-groupXX.less.

Jeff

**Comment by dominikwilkowski** [comment]
Hey Jeff,

Ok step by step:
> One other thing is that should the Blender also make a gui.less file available?

That's a great idea. Can I ask you to launch a different issue so we can track it separately and close it off with the a commit?

> Also Dominik, when an issue is marked as closed and committed, when will the Blender be updated such that the new LESS files will be available in the download? Is this immediate integration or does a release have to be planned to update the Blender?

As the GUI website and the GUI source are two different repositories this will always take a bit of time. I have to commit to this repo *and* then update the submodule in the other repo to include the change *and* built the docs out *and* upload them.
TL;DR: I will upload them in a minute though to make it short :)

> Had a look on GitHub and could not find the files icon-groupXX.less.

Just to reiterate, only the module icon-group01 has css. All other icon-groups are just svgs so you wouldn't find anything there.

Also I do get notified when you add a comment even to a closed issue. We encourage discussion here :+1:

**Comment by dominikwilkowski** [comment]
up now @jeffreyselby
